### ISSUE TITLE
Inconsistent Highlighting of Code on CSS Page

### ISSUE THREAD

**Issue by loganfranken** [UCSantaBarbara/webguide#210]
The CSS page sometimes highlights code words (!important, div) and sometimes doesn't:
http://webguide.ucsb.edu/css/

**Comment by garster** [comment]
@loganfranken can you provide screenshots?

**Comment by loganfranken** [comment]
Hey @garster!

Here's all of the words that I _think_ should be highlighted as code words but aren't (although I could have missed a few):

![image](https://cloud.githubusercontent.com/assets/1024444/26600838/70571d88-4532-11e7-958a-bc96b3b19380.png)

![image](https://cloud.githubusercontent.com/assets/1024444/26600843/74503672-4532-11e7-9df9-d817f2c4cb71.png)

![image](https://cloud.githubusercontent.com/assets/1024444/26600860/84a861a2-4532-11e7-9e9f-b42ad695d14e.png)

![image](https://cloud.githubusercontent.com/assets/1024444/26600879/985d4e88-4532-11e7-8680-d579fe4e7b66.png)

![image](https://cloud.githubusercontent.com/assets/1024444/26600887/9c59820e-4532-11e7-92da-418decef60c9.png)

![image](https://cloud.githubusercontent.com/assets/1024444/26600902/a487a406-4532-11e7-991d-d4abffeaf1da.png)

![image](https://cloud.githubusercontent.com/assets/1024444/26600913/b0e2c8b6-4532-11e7-93c0-758fc8d9ff1b.png)

![image](https://cloud.githubusercontent.com/assets/1024444/26600929/c046f32c-4532-11e7-9dcc-38563717c61a.png)

![image](https://cloud.githubusercontent.com/assets/1024444/26600952/d67da924-4532-11e7-9ac0-6bc599141a3c.png)

![image](https://cloud.githubusercontent.com/assets/1024444/26600963/e3b9f7a0-4532-11e7-8f0b-2acddd0bb15d.png)

**Comment by garster** [comment]
@loganfranken I did some work but I am unsure about `div`. In some places it is plural, so how would that be handled. Also there are references to paragraphs. So yes div is a tag, it seems to me div and paragraph are both referring to elements and should be treated the same.

I will do some more tweaking on this before PR.

**Comment by loganfranken** [comment]
Hey @garster! Personally, for plural "div", I would just go with `div`s (but I'm open to whatever).

I usually reserve the code highlighting for when it's actually code. In other words, I highlight "div" because it's an actual HTML element. I _wouldn't_ highlight "paragraph", but I _would_ highlight "p".

(But those are just my personal preferences, of course)

**Comment by garster** [comment]
@loganfranken I saw the "`div`s" once in Bootstrap docs and to me it looks like a typo. I noticed more and more they are using the tag with brackets `<div>` and thus `<div>`s. They use . and # for classes and ids respectively. 

> To make the jumbotron full width, and without rounded corners, place it outside all `.container`s and instead add a `.container` within. (http://getbootstrap.com/components/#jumbotron)

I think using the brackets helps when you get to single digit tag name like `a` and `p` and it my guess why division is written as div but paragraph is NOT written as p. Using this method we could standardize references to all html tags, such as `<p>`.

I like the idea of using the full tag and the dot, hash when referencing html tags, classes and ids.

- `<p>` `<div>`s
- `.my-class` `.my-stuff`s
- `#item` `#my-id`s

Sounds like we need a writing style guide to guide authors to consistent content.
